package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "export": "next build && next export"
```

```suggestion
    "export": "next export"
```

export only should be enough -

---

Actually no, my bad: https://nextjs.org/docs/advanced-features/static-html-export

---

as per the doc it should be `"export": "next build && next export"`